pkgs/development/python-modules/hg-git/default.nix

```diff
@@ -3,34 +3,28 @@
 , fetchPypi
 , dulwich
 , isPy3k
+, stdenv
 , fetchpatch
+, brotli
 }:
 
 buildPythonPackage rec {
   pname = "hg-git";
-  version = "0.8.12";
-  disabled = isPy3k;
+  version = "0.9.0";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "13hbm0ki6s88r6p65ibvrbxnskinzdz0m9gsshb8s571p91ymfjn";
+    sha256 = "0wbnhvn0c9w1hg0ykkqh7vqzr48sfgh33na6j08fqir86a6bvb3p";
   };
 
-  propagatedBuildInputs = [ dulwich ];
+  propagatedBuildInputs = [ dulwich brotli ];
 
-  # Needs patch to work with Mercurial 4.8
-  # https://bitbucket.org/durin42/hg-git/issues/264/unexpected-keyword-argument-createopts-hg
-  patches =
-    fetchpatch {
-      url = "https://bitbucket.org/rsalmaso/hg-git/commits/a778506fd4be0bf1afa75755f6ee9260fa234a0f/raw";
-      sha256 = "12r4qzbc5xcqwv0kvf8g4wjji7n45421zkbf6i75vyi4nl6n4j15";
-    };
+  doCheck = false;
```

Comment:
Can you add a comment why you disabled tests?

Reply:
commented, the tests require mercurial to be installed, and mercurial has a hg-git dependency now.

You think we need to specify a hg-git-free mercurial derivation and perform the tests?

Reply:
> You think we need to specify a hg-git-free mercurial derivation and perform the tests?

If you can easily do it then you can try to achieve it but tests are not a hard requirement.

Reply:
If it isn't I would maybe prefer to defer it to another PR and get that change "in" for now.